protos/platform/v0/platform.proto

```diff
@@ -85,3 +86,15 @@ message GetIdentityIdsByPublicKeyHashesResponse {
   repeated bytes identity_ids = 1;
   Proof proof = 2;
 }
+
+message StateTransitionResultsRequest {
+  repeated bytes state_transition_hashes = 1;
+  bool prove = 2;
+}
+
+message StateTransitionResultsResponse {
+  oneof responses {
+    string errors = 1;
+    Proof proof = 2;
```

Comment:
Do we send only proofs or data as well?

Reply:
We need to send ST hash, but not the data. The point of proving something is that client got back what he was expecting to get. The verification consists of two steps: 
1. Client verifies that he got what he expected to get - that the server indeed responded with a full proof for the data in the ST, i.e. the client needs to reconstruct the data hashes
2. Verify the hashes with the proof

The only case when the data can be useful for the client is to check that the data from the response matches what the client has in its local state. But that is redundant since the hashes of the data are already included in the proof

Reply:
Make sense!